shell/platform/fuchsia/flutter/kernel/libraries.json

```diff
@@ -1,163 +1,160 @@
 {
   "comment:0": "NOTE: THIS FILE IS GENERATED. DO NOT EDIT.",
```

Comment:
Should we remove this line?  Is it true anymore?

Reply:
yeah, we can still generate this using `libraries.yaml` i haven't verified, but in theory it should just work.

Reply:
fwiw, i've also committed the changes to `libraries.yaml` in this PR